.env.example

```
@@ -1,3 +1,5 @@
 VITE_SERVER_URL=
 VITE_APP_GUEST_LOGIN=
-VITE_APP_GUEST_PASSWORD=
\ No newline at end of file
+VITE_APP_GUEST_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_APP_GUEST_PASSWORD key should go before the VITE_SERVER_URL key

(UnorderedKey)